logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/asset/ScheduleNewAssetMessageUseCase.kt

```diff
@@ -199,9 +280,16 @@ internal class ScheduleNewAssetMessageUseCaseImpl(
                 name = assetName,
                 mimeType = mimeType,
                 metadata = when {
-                    isDisplayableImageMimeType(mimeType) && (assetHeight.isGreaterThan(0) && (assetWidth.isGreaterThan(0))) -> {
+                    isDisplayableImageMimeType(mimeType) && (
+                            assetHeight.isGreaterThan(0) && (
+                                    assetWidth.isGreaterThan(
+                                        0
+                                    )
+                                    )
+                            ) -> {
```

Comment:
Can we extract this to a separate function? Or format on a different way:
```suggestion
                   isDisplayableImageMimeType(mimeType) && (assetHeight.isGreaterThan(0) && (assetWidth.isGreaterThan(0))) -> {
                        AssetContent.AssetMetadata.Image(assetWidth, assetHeight)
                    }
```

This way is really hard to read.